framework/aster-frame/src/sync/wait.rs

```diff
@@ -127,6 +99,7 @@ impl WaitQueue {
         }
     }
 
+    /// Return if current waitqueue is empty.
```

Comment:
- `if` is really ambiguous here. Use `whether` instead.
 - `waitqueue` -> `wait queue`.
 
Perhaps also fix comments of other methods in `WaitQueue`.

Actually, this method is racy by design, so I think that it should be removed. But that's obviously outside the scope of this PR.